### DISCUSSION TITLE
will AI take over coding jobs in the near future ???

### DISCUSSION THREAD

**Discussion by LaptopJock** [Cyfrin/foundry-full-course-cu#2121] [Polls]
with AI booming, do you guys think coding is safe or will AI replace us in those jobs too ??

**Comment by Oscar-Grouch-Token** [comment]
It will make our job easier and you will see more people pour into this space.

**Comment by Oscar-Grouch-Token** [comment]
![Ad1](https://github.com/user-attachments/assets/ce0566aa-5f64-4bd9-ab0c-ff0dac87ae79)
I show you this because we are getting the kids excited about learning blockchain...

**Reply by Oscar-Grouch-Token** [reply]
This is part of a project I am working on.

**Reply by EngrPips** [reply]
Interesting.

**Reply by Oscar-Grouch-Token** [reply]
https://sites.google.com/view/oscar-grouch-token-ogt/home  --- This will be a good educational project for kids and adults of all ages. I look forward to the up and coming uses for this tech. I have so many projects in mind. I look forward to my future in blockchain. I want to teach everyone about the concept.

**Reply by EngrPips** [reply]
I wish you the best in the Blockchain space.

**Comment by nikolov9996** [comment]
Already clearing tech debt from vibe coders... just more works ahead

**Reply by EngrPips** [reply]
Keep crushing it.
